libs/tools/src/executors/release/build-update-publish/executor.spec.ts

```diff
@@ -19,7 +18,6 @@ describe('BuildUpdatePublish Executor', () => {
 		jest.spyOn(projectHelper, 'getProjectName').mockReturnValue(libName);
 
 		// Mock updateVersion and npmPublish to return { success: true }
```

Comment:
The comment is now confusing when updateVersion is not here anymore.

Reply:
Good point! I'll remove it